### PR TITLE
Feature/minor issues payconiq

### LINKED ISSUE #856: Minor issues Payconiq
Since the release of payconiq some issues have been discovered that should be fixed before the next bookyear.

- [ ] Activity length shouldn't be over 140 characters for Exact
- [ ] First activity name then Member ID (I.e "Airco: Antwerpen - 1234" instead of "1234 - Airco: Antwerpen"
- [ ] Mongoose pinned transactions seems to be broken
- [ ] Export transactions should be clearer on what date range you get
- [ ] Settings display incorrect values

### PR DESCRIPTION
**Pull request by JustNireon**
resolves #856 
Also reworked the member payment page to make it more accessible.
Example:
image

Should be noted that there is a migration to limit the amount of characters to 52 since this is the limit of Exact descriptions. Shouldn't an issue since no activity should be longer than 52 characters.

### CODE REVIEW

File: config/locales/en.yml [hunk 1]
```diff
@@ -62,7 +62,7 @@ en:
         category: Category
         image: Image
         name: Name
-        price: Price
+        price: PriceNaam
```
Comment (Riscky) [inline config/locales/en.yml:65]: I don't think this is correct?

File: config/locales/members.en.yml [hunk 2]
```diff
@@ -94,8 +94,24 @@ en:
       pictures: Pictures
       vacancies: Vacancies
     payments:
+      mongoose:
+        credit: Credit
+        subtitle: You can solely use mongoose credit to pay for product in Sticky's room.
```
Comment (Riscky) [inline config/locales/members.en.yml:99]: You can **only** use **M**ongoose credit to pay for product**s** in Sticky's room.

File: config/locales/members.en.yml [hunk 3]
```diff
@@ -94,8 +94,24 @@ en:
...
+        transaction_costs: transaction costs charged
```
Comment (Riscky) [inline config/locales/members.en.yml:100]: start with capital

File: config/locales/members.nl.yml [hunk 4]
```diff
@@ -75,13 +75,13 @@ nl:
         study: Studiegegevens
         underage_phone: Als je minderjarig bent is het verplicht om een telefoonnummer in te vullen voor noodgevallen.
       index:
-        credit: Tegoed
+        credit: Credit
         edit_profile: Gegevens bewerken
-        mongoose-info: Mongoose tegoed is niet voor het betalen van activiteiten of lidmaatschap. Mongoose kan je uitsluitend gebruiken om in de sticky gezelligheidskamer producten te kopen.
+        mongoose-info: Mongoose credit is not to pay for activities or membership. You can solely use mongoose credit to pay for product in Sticky's room.
```
Comment (Riscky) [inline config/locales/members.nl.yml:80]: Dit is de NL file, terugveranderen naar nederlands gerne :)

File: config/locales/members.nl.yml [hunk 5]
```diff
@@ -94,8 +94,24 @@ nl:
       pictures: Foto's
       vacancies: Vacatures
     payments:
+      mongoose:
+        credit: Tegoed
+        subtitle: Mongoose kan je uitsluitend gebruiken om in de sticky gezelligheidskamer producten te kopen.
```
Comment (Riscky) [inline config/locales/members.nl.yml:99]: **S**ticky

File: db/migrate/20210808184948_limit_activity_characters.rb [hunk 6]
```diff
@@ -0,0 +1,11 @@
+class LimitActivityCharacters < ActiveRecord::Migration[6.0]
+  def up
+    Activity.where('length(name) > 52').each {|activity|
```
Comment (Riscky) [inline db/migrate/20210808184948_limit_activity_characters.rb:3]: geen down migratie, terwijl dat wel vrij makkelijk is (limit terugveranderen, activiteitnamen gelijkhouden)

File: app/views/members/payments/index.html.haml [hunk 7]
```diff
@@ -10,69 +10,121 @@
...
   .col-lg-6.col-12
     .card
       = form_with :url => :mongoose, :local => true, :method=> :post, :class => 'form-validation', :id => "add_funds_form" do |f|
         .card-header
-          %i.fa.fa-fw.fa-fw.fa-shopping-cart
-          %span= I18n.t("activerecord.models.checkout_transactions")
-          = f.select :payment_type, [["Payconiq +€#{String(@payconiq_transaction_costs)}", 'Payconiq'],["Ideal +€#{String(@transaction_costs)}", 'Ideal']], {}, { :id => 'payment_type_add_funds' }
-          %table.table.card-body
+          .mb-2
+            %i.fa.fa-fw.fa-fw.fa-shopping-cart
```
Comment (Riscky) [inline app/views/members/payments/index.html.haml:97]: dubbele `fa-fw`

**Review by Riscky — CHANGES_REQUESTED** [review]
Really happy with this PR :)
Lots of small comments, so please go through your code thoroughly.

File: config/locales/members.nl.yml [hunk 8]
```diff
@@ -75,13 +75,13 @@ nl:
...
       index:
-        credit: Tegoed
+        credit: Credit
         edit_profile: Gegevens bewerken
         mongoose-info: Mongoose tegoed is niet voor het betalen van activiteiten of lidmaatschap. Mongoose kan je uitsluitend gebruiken om in de sticky gezelligheidskamer producten te kopen.
         outstanding_payments: Openstaande betalingen
         personal_data: Persoonsgegevens
-        transaction_costs: transactiekosten in rekening gebracht
-        upgrade: Opwaarderen
+        transaction_costs: transaction costs charged
```
Comment (Mstiekema) [inline config/locales/members.nl.yml:83]: Reden dat dit van Nederlands naar Engels is veranderd?

File: db/seeds/payments.rb [hunk 9]
```diff
@@ -5,7 +5,7 @@
 
 Member.all.sample(30).each do |member|
 
-  5.times do
+  15.times do
```
Comment (Mstiekema) [inline db/seeds/payments.rb:8]: 'vo

**Review by Mstiekema — CHANGES_REQUESTED** [review]
Looking good, please look at the two relevant comments and fix if needed :)

**Review by Mstiekema — APPROVED** [review]
Lgtm :)

**Pull request merged**